src/deploymentinfo.cpp

```diff
@@ -89,6 +93,8 @@ const std::map<std::string, uint32_t> g_verify_flag_names{
     FLAG_NAME(DISCOURAGE_CHECK_TEMPLATE_VERIFY_HASH)
     FLAG_NAME(ANYPREVOUT)
     FLAG_NAME(DISCOURAGE_ANYPREVOUT)
+    FLAG_NAME(TAPSCRIPT_OP_CAT)
+    FLAG_NAME(DISCOURAGE_TAPSCRIPT_OP_CAT)
```

Comment:
Is there a reason not to change this to?
`TAPSCRIPT_OP_CAT` --> `OP_CAT`
`DISCOURAGE_TAPSCRIPT_OP_CAT` --> `DISCOURAGE_OP_CAT`

It doesn't look like any of the other flags use the `TAPSCRIPT` prefix.